be/src/exec/vectorized/olap_scan_node.cpp

```diff
@@ -66,7 +66,7 @@ Status OlapScanNode::open(RuntimeState* state) {
     _update_status(status);
 
     _dict_optimize_parser.set_mutable_dict_maps(state->mutable_global_dict_map());
-    DictOptimizeParser::rewrite_descriptor(state, _tuple_desc->slots(), _conjunct_ctxs,
+    DictOptimizeParser::rewrite_descriptor(state, _tuple_desc->decoded_slots(), _conjunct_ctxs,
```

Comment:
The second argument of DictOptimizeParser::rewrite_descriptor() is vector, I think it is better to use a const reference. And the third argument is better to be changed to pointer from reference.

Reply:
Fix